plat/st/stm32mp1/stm32mp1_security.c

```diff
@@ -76,3 +116,12 @@ void plat_arch_security_setup(void)
 {
```

Comment:
similar comment as previous regarding the `plat_` prefix.

Reply:
OK